Modific.py

```diff
@@ -784,3 +784,18 @@ def open_files(self, *files):
                     self.window.open_file(os.path.join(self.vcs['root'], fname))
                 else:
                     sublime.status_message("File '{0}' doesn't exist".format(fname))
+
+
+class ToggleHighlightChangesCommand(sublime_plugin.TextCommand):
+    def run(self, edit):
+        v_setting = "highlight_changes"
+        settings = get_settings()
+
+        if settings.get(v_setting):
+        	settings.set(v_setting, False)
+        	self.view.run_command('save')
+        else:
+        	settings.set(v_setting, True)
+        	self.view.run_command('save')
```

Comment:
Let's get rid of that if-else, and remove regions instead of saving the file.

```py
settings.set(v_setting, bool(settings.get(v_setting)))
if not settings.get(v_setting):
    # remove highlighting
    map(self.view.erase_regions, ('inserted', 'changed', 'deleted'))
```